src/server.py

```diff
@@ -54,6 +55,8 @@ def __init__(self, bus_name, sound_event_id, uuid_, metadata,
     def release(self):
         # Otherwise, GStreamer complains with a WARNING indicating that
         # g_idle_add should be used.
+        if self._error:
```

Comment:
This new code seems to separate the above comment from the code that it refers to. It would be better to move the comment down.